tests/test_resources.py

```diff
@@ -14,3 +14,15 @@ def test_all_resources(api_client):
     assert len(resources) == 2
     for resource in resources:
         assert isinstance(resource, RoomResource)
+
+
+@pytest.mark.usefixtures("mock_resources")
+def test_resource(api_client):
```

Comment:
@yusra-ahmed It's not much but the endpoint is a very basic `GET` endpoint without an query support!